README.md

```diff
@@ -1,6 +1,11 @@
 ![Build Status](https://github.com/ebachelet/pyLIMA/actions/workflows/actions_unit_tests.yaml/badge.svg)
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.997468.svg)](https://doi.org/10.5281/zenodo.997468)
 
+
+# WARNING
+
+We are aware of crashes when pyLIMA is called with python3.11+ using MAC (and potentially Windows). This is due to the multiprocessing module and we are currently working on a fix. Meanwhile, please try to use lower version of python with these OS.
```

Comment:
I recommend providing a bit more detail in the warning, and also I think we shouldn't necessarily phrase it as a warning - after all, you implemented this to speed the code up and it works fine under Linux.  Here's some suggested text.  

# Multiprocessing in PyLIMA

The latest version of PyLIMA applies the [multiprocessing library](https://docs.python.org/3/library/multiprocessing.html#the-spawn-and-forkserver-start-methods) to parallelize aspects of its model fitting processes in order to optimize for speed.  This has been tested and works under Ubuntu Linux with Python 3.11.  
Users should be aware that the multiprocessing library uses a different method ('spawn') to start threads on the Mac and Windows platforms compared to the method used on Linux ('fork'), as the fork method is considered to be unsafe on these platforms.  Unfortunately, this has meant that PyLIMA crashes if run under the latest version of Python (3.11) under a Mac (and likely under Windows), due to [the outstanding issue](https://github.com/python/cpython/issues/102205) with the multiprocessing library.  
We are currently investigating a fix for this issue.  In the interim we recommend using an earlier version of Python with the latest PyLIMA.